src/app.js

```diff
@@ -56,7 +56,25 @@ export function app(app) {
     })
   }
 
+  function hydrate(elm) {
```

Comment:
Not certain I understand what you mean but changes/improvements welcome and if you think there is a better way to do this then I am all for it.